topo/node/ceos/ceos.go

```diff
@@ -135,16 +135,29 @@ func (n *Node) GenerateSelfSigned(ctx context.Context) error {
 		),
 	}
 
-	resp, err := n.cliConn.SendConfigs(cfgs)
-	if err != nil {
-		return err
-	}
-
-	if resp.Failed == nil {
-		log.Infof("%s - finshed cert generation", n.Name())
+	for pkiReady := false; !pkiReady; {
```

Comment:
pkiReady := false
for !pkiReady {
...
}

Reply:
Fixed